src/ecmascript_simd.js

```diff
@@ -474,13 +474,13 @@ if (typeof SIMD.Bool32x4.select === "undefined") {
   }
 }
 
-if (!Object.hasOwnProperty(SIMD.Bool32x4.prototype, 'valueOf')) {
+if (!SIMD.Bool32x4.prototype.hasOwnProperty('valueOf')) {
```

Comment:
What do we do to prevent someone from shadowing Object.prototype.hasOwnProperty? Would this really be any better?